src/mvs_eland/E1_process_results.py

```diff
@@ -161,7 +161,7 @@ def get_storage_results(settings, storage_bus, dict_asset):
 
     if OPTIMIZE_CAP in dict_asset:
         if dict_asset[OPTIMIZE_CAP][VALUE] is True:
-            power_charge = storage_bus[KPI_SCALARS_DICT][
+            power_charge = storage_bus['scalars'][
```

Comment:
going from variable back to string?

Reply:
Intentional, as this may not be changed!